tools/migrate/setup.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import sys
+from setuptools import setup
+
+setup(name='migrate',
+      description='Create and apply alembic migrations',
+      url='http://hystax.com',
+      author='Hystax',
+      author_email='info@hystax.com',
+      package_dir={'migrate': ''},
+      packages=['migrate'],
```

Comment:
Probably `db` for the name of the package and `migrations` for the command name it's a bit better so you can run it as `python -m tools.db.migrations generate` or `python -m tools.db.migrations migrate`.